Tutorials/HowToUseMonteCarloSimulation.md

```diff
@@ -85,6 +85,14 @@
   ```c++
   #include "MCSimExecutor.h"
   ```
+- If you find description below
+  ```c++
+  #include "Interface/LogOutout/Logger.h"
+  ```
+  rewrite as the following description 
```

Comment:
The sample codes in the [MCSim](https://github.com/ut-issl/s2e-documents/blob/main/Tutorials/SampleCodes/MCSim/S2eUser.cpp) directory, you can find the correct `S2eUser.cpp` which uses `InitLog.hpp`.  
But I agree with the description is confusing. Could you modify the sample code in [s2e_user](https://github.com/ut-issl/s2e-documents/blob/main/Tutorials/SampleCodes/s2e-user/src/S2eUser.cpp) to include the `InitLog.hpp`?

Reply:
I've dealt with it. Should I reset the changes (add description) in the document?

Reply:
No, you don't need to revert this document. The added descriptions are better for easy understanding.